triagebot.toml

```diff
@@ -230,6 +240,24 @@ trigger_files = [
     "src/tools/jsondoclint",
 ]
 
+[autolabel."T-rustdoc-frontend"]
```

Comment:
Shouldn't there be `trigger_files` here too?

Reply:
I *think* this is fine because it will transitively trigger due to the `A-*` labels themselves getting triggered on files.

Reply:
I think it's not great to rely on transitive triggers, especially when we remove a "parent". Then children won't trigger it anymore and we'll only discover it anymore. Would be nice if we could say that a label depends on another directly.

Reply:
Yeah, probably safer to duplicate the relevant path filters here.

Reply:
I know but since I'm not the owner here, I'll wait for them to update. If they don't want to, I can always send a follow-up.

Reply:
My biggest concern is that T-rustdoc and T-rustdoc-frontend are mutually exclusive, and the paths that would trigger the frontend label are in dirs that trigger the T-rustdoc label.

i'm not sure if the longer path would take precedent, or if it would depend on hashmap ordering, or what

Reply:
I thought they would all be triggered...

Reply:
oh, that's an option i hadn't considered, adding both T-rustdoc and T-rustdoc-frontend, even though they're supposed to be mutually exclusive...

hmm... i suppose it's unlikely to be anything destructive, so if it does cause an issue, it should be pretty easy to fix...

Reply:
after reading through [the relevant code](https://github.com/rust-lang/triagebot/blob/d7d4babe564e264ea4824bba1ba55a8088d8a9fd/src/handlers/autolabel.rs), it appears that all autolabel rules apply simultaneously, so it would indeed add both.  I'll add the trigger_files